src/nxdoc/web-ui/web-ui-functional-testing.md

```diff
@@ -160,10 +160,12 @@ For an example of how to launch a Nuxeo server and run the tests using maven, pl
 ### Debugging
 
 You can run the previous commands in debug mode by passing the `--debug` parameter:
+For debugging purposes, you can use this command:
 
 ```
-npm run test -- --debug
-npm run test:watch -- --debug
+npm run ftest:dev
```

Comment:
the ftest:watch can also be used for debugging, right?

Reply:
I mention that in the next line. Do you think it is confusing?

Reply:
I believe that just listing one Or the other would make it simple to understand.